packages/core/src/external/carequality/ihe-gateway-v2/outbound/xca/create/iti39-envelope.ts

```diff
@@ -10,6 +10,14 @@ import { OutboundDocumentRetrievalReq, XCAGateway } from "@metriport/ihe-gateway
 
 const action = "urn:ihe:iti:2007:CrossGatewayRetrieve";
 
+const onlyOneDocumentReferencePerRequestList = [
+  "2.16.840.1.113883.3.6448",
+  "2.16.840.1.113883.3.6147.458",
+  "2.16.840.1.113883.3.6147.458.2",
```

Comment:
nit: would be useful to put these in consts so we can refer to them later as needed - ie
```
const redoxGatewayOid = "2.16.840.1.113883.3.6448";
const onlyOneDocumentReferencePerRequestList = [
  redoxGatewayOid,
  ...
```

nit: `onlyOneDocumentReferencePerRequestList` isn't very semantic - hard to understand what it does at a glance

perhaps `gatewaysThatAcceptOneDocRefPerRequest` would be better?